tracer/src/Datadog.Trace/Vendors/Microsoft.OpenApi/Services/OpenApiUrlTreeNode.cs

```diff
@@ -273,11 +273,15 @@ private static void ProcessNode(OpenApiUrlTreeNode node, TextWriter writer)
         {
             var path = string.IsNullOrEmpty(node.Path) ? "/" : SanitizeMermaidNode(node.Path);
             var methods = GetMethods(node);
-            var (startChar, endChar) = GetShapeDelimiters(methods);
+            var tuple = GetShapeDelimiters(methods);
+            var startChar = tuple.Item1;
+            var endChar = tuple.Item2;
```

Comment:
Wow, that's annoying 😅 We should still encode this in the vendoring tool somehow though 🙁

Reply:
I started working on it, it was super annoying, and then I realized I could just delete it all :D